README.md

```diff
@@ -1,3 +1,7 @@
+[![CharmHub Badge](https://charmhub.io/nginx-ingress-integrator/badge.svg)](https://charmhub.io/nginx-ingress-integrator)
+[![Publish to edge](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/test_and_publish_charm.yaml/badge.svg)](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/test_and_publish_charm.yaml)
```

Comment:
Will this show the last release? Perhaps we can rename "Publish to edge" with "Edge release" or something similar?
Can we also include another bad for stable, the promote charm one?

Reply:
Good point, I didn't notice that. 
- I'll change it to "Release to Edge" so will be like the existing ones here:
https://releases.juju.is/?team=Observability
- Promote charm added